partner_communication/models/communication_job.py

```diff
@@ -129,8 +129,9 @@ def count_pdf_page(self):
         if not skip_count:
             for record in self.filtered('report_id'):
                 if record.send_mode == 'physical':
+                    lang = self.env.context.get('lang', record.partner_id.lang)
                     report_obj = record.env['report'].with_context(
-                        lang=record.partner_id.lang,
+                        lang=lang,
                         must_skip_send_to_printer=True)
                     pdf_str = report_obj.get_pdf(record.ids,
```

Comment:
Changement pour qu'une communication job suivent la langue du context s'il y a en a une

Reply:
La même fonctionnalité a été ajouter à différents endroits dans le code

Reply:
I think you should remove this change. The problem is that the context contains the language of the user, and we don't want to print letters in the language of the connected user, but in the language of the sponsor instead.